test/common/globalBefore.js

```diff
@@ -1,8 +1,14 @@
 'use strict';
```

Comment:
I'd like to have a consistent order in this file. Functions at the top, later vars and prototypes at the bottom.